Makefile

```diff
@@ -71,10 +71,12 @@ deps\:lint:
 
 .PHONY: deps\:test
 deps\:test:
+	rustup target add wasm32-unknown-unknown
 	go install gotest.tools/gotestsum@latest
 
 .PHONY: deps\:coverage
 deps\:coverage:
+	rustup target add wasm32-unknown-unknown
```

Comment:
todo: This is not a coverage dependency (this should be scoped to tools needed for `running` code coverage generation).

Reply:
It is at the moment, but there is a task in the description to rework this based off the `make test:lens` discord discussion.  This line will be removed